### PR TITLE
Update Nix env

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -71,6 +70,8 @@
               src = craneLib.cleanCargoSource (craneLib.path ./.);
               doCheck = false;
 
+              cargoExtraArgs = "-p trycp_server";
```
Comment (ThetaSinner) [inline flake.nix:73]: Select only what we want to build. It was actually building wasms too and creating a larger bundle than necessary

**Review by jost-s — APPROVED** [review]
bang on!

**Comment by ThetaSinner** [comment]
Of course this doesn't build because of DPKI...

**Comment by ThetaSinner** [comment]
We really need that other PR finished off

**Pull request closed without merging**